pdq/cpp/bin/benchmark-query.cpp

```diff
@@ -0,0 +1,288 @@
+#include <stdio.h>
```

Comment:
bocking q: Is this just from your other PR? Can you rebase now that it is merged and only include the content from this changeset?